content/transactions/checkcreate.md

```diff
@@ -0,0 +1,41 @@
+## CheckCreate
+[[Source]<br>](https://github.com/ripple/rippled/blob/master/src/ripple/app/tx/impl/CreateCheck.cpp "Source")
```

Comment:
I'm getting a `404` for this link. I assume that's because it's valid on `develop`, but just checking that was your intention.

https://github.com/ripple/rippled/blob/develop/src/ripple/app/tx/impl/CreateCheck.cpp

Reply:
As above, temporarily pointed to develop until the code is released